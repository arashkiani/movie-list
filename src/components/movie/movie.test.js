/*global shallow, mount, sinon */
import Component from '.'
const movie = {title:'title', overview:'overview', poster_path:'/poster_path', backdrop_path:'/backdrop_path', vote_average:1}
let shallowWrapper

beforeEach(async () => {
	shallowWrapper = shallow(<Component {...movie}/>)
})
describe('<Movie />', () => {
	it('Match Snapshot', () => {
		expect(shallowWrapper).toMatchSnapshot()
	})
	it('Props pass movie img url && title into component', () => {
		expect(shallowWrapper.find('img').props().src).toBe('https://image.tmdb.org/t/p/w185_and_h278_bestv2/poster_path')
		expect(shallowWrapper.find('h2').text()).toBe(movie.title)
	})
})