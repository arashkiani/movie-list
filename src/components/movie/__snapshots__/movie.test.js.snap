// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Movie /> Match Snapshot 1`] = `
<ForwardRef>
  <ForwardRef>
    <img
      alt="title Poster Image"
      src="https://image.tmdb.org/t/p/w185_and_h278_bestv2/poster_path"
    />
  </ForwardRef>
  <ForwardRef>
    <ForwardRef>
      <h2>
        title
      </h2>
      1
    </ForwardRef>
    <ForwardRef>
      overview
    </ForwardRef>
  </ForwardRef>
</ForwardRef>
`;
